src/main/kotlin/com/hibob/academy/filter/AuthenticationFilter.kt

```diff
@@ -30,12 +30,12 @@ class AuthenticationFilter: ContainerRequestFilter {
             try {
                 Jwts.parser().setSigningKey(SECRET_KEY).parseClaimsJws(it)
             } catch (e: Exception) {
-                extracted(requestContext, e)
+                abortUnauthorized(requestContext, e)
```

Comment:
This change is not relevant here
It can be solved by rebase

Reply:
Changed it back and opened a new branch and did it there (waiting for Eli).
Fixed